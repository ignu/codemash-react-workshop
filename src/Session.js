import React, { Component } from 'react';

class Session extends Component {
  render() {
    return (<div>
      <div>{this.props.title}</div>
      <div>{this.props.abstract}</div>
    </div>
           )
  }
}

Session.propTypes = {
  title: React.PropTypes.string.isRequired,
  abstract: React.PropTypes.string.isRequired
}

export default Session
