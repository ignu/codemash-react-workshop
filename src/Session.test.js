import React from 'react';
import Session from './Session';
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const output = renderer.create(<Session title="a" abstract="b" />);

  expect(output).toMatchSnapshot()
});
