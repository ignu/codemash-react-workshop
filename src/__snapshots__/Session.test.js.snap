exports[`test renders without crashing 1`] = `
<div>
  <div>
    a
  </div>
  <div>
    b
  </div>
</div>
`;
